### DISCUSSION TITLE
`libexpat1-dev` dependency of libvips

### DISCUSSION THREAD

**Discussion by LoyalPotato** [libvips/libvips#3930] [Q&A] [answered]
Hello,

Why does `libvips` depend on this `libexpat1-dev` (for example) but in this [Dockerfile example](https://github.com/jcupitt/docker-builds/blob/master/libvips-amazonlinux2/Dockerfile) its installing `expat-devel` instead. Is this because of the underlying OS?

I realize that this question might not be really specific to libvips, but I don't know how to search for an answer.

**Comment by jcupitt** [comment, marked as answer]
Hello @LoyalPotato,

Debian-style linux systems generally call this something like `libexpat-dev`, redhat-derived linuxes use `expat-devel`. Other linuxes have their own strange systems.

**Reply by LoyalPotato** [reply]
Thank you!
